Add logo usage rights explainer (#3109)
sourcegraph/handbook: content/departments/marketing/brand/brand_guidelines/logo.md

### Wordmark use

See [PDF](https://sourcegraphstatic.com/Sourcegraph_Brand_Guidelines.pdf#page=23)

## Logo usage rights

> Displaying the Sourcegraph logo on your website or other digital asset is only allowed with express permission and usually governed through the vendor contract (refer to [Sourcegraph's procurement policy](/departments/finance/process/ap/#commercial-negotiations)).

Author: ```suggestion
Displaying the Sourcegraph logo on your website or other digital asset is only allowed with express permission and usually governed through the vendor contract (refer to [Sourcegraph's procurement policy](/departments/finance/process/ap.md#commercial-negotiations)).
```

> Displaying the Sourcegraph logo on your website or other digital asset is only allowed with express permission and usually governed through the vendor contract (refer to [Sourcegraph's procurement policy](/departments/finance/process/ap.md#commercial-negotiations)).

Author: ```suggestion
Displaying the Sourcegraph logo on your website or other digital asset is only allowed with express permission and usually governed through the vendor contract (refer to [Sourcegraph's procurement policy](../../../finance/process/ap.md#commercial-negotiations)).
```

> Startup companies, who's product or service we use and who we don't have a formal vendor contact with, that would like to showcase the Sourcegraph logocan request approval as long as their company conforms to the general criteria of a startup: VC-backed, pre-IPO and under 200 people employed. Written permission can be obtained from the primary contact for the account.

Reviewer: replace "who's" with "whose"

> Startup companies, who's product or service we use and who we don't have a formal vendor contact with, that would like to showcase the Sourcegraph logocan request approval as long as their company conforms to the general criteria of a startup: VC-backed, pre-IPO and under 200 people employed. Written permission can be obtained from the primary contact for the account.

Reviewer: remove "and who we don't have a formal vendor contract with" 
reason for removal: we should have a vendor contract with every company whose product/service we use, even if it's online terms, so let's avoid condoning using vendor services without a formal vendor contract.

> Startup companies, who's product or service we use and who we don't have a formal vendor contact with, that would like to showcase the Sourcegraph logocan request approval as long as their company conforms to the general criteria of a startup: VC-backed, pre-IPO and under 200 people employed. Written permission can be obtained from the primary contact for the account.

Author: ```suggestion
Startup companies, whose product or service we use, that would like to showcase the Sourcegraph logo can request approval as long as their company conforms to the general criteria of a startup: VC-backed, pre-IPO, and under 200 people employed. Written permission can be obtained from the primary contact for the account.
```
